.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+### Please consider the following when opening a pull request:
+
+- Link to the relevant JIRA ticket(s) and tag any relevant team(s).
+- Squash your changes down into one or more discrete commits.
+  In each commit, include description that could help a developer
+  several months from now.
+- If running `make upgrade`, run _as close to the time of merging as possible_
+  to avoid accidentally downgrading someone else's package.
+- Aim for comprehensive test coverage, but remember that
+  automated testing isn't a substitute for manual verification.
+- Carefully consider naming, code organization, dependencies when adding new code.
```

Comment:
Add a link to our top-level ADR directory as a reference for edx-platform-wide decisions?  https://github.com/edx/edx-platform/tree/master/docs/decisions